File: C/PRAK104-2210817110005-MIZZUDDINABDISSALAM.c
#include <stdio.h>

int main(){
    int sepA, SepB, Dis13 = 13, Dis21 = 21;
    int hasil1, hasil2;
    char d = '%';
    printf("harga sepatu A adalah :");
    scanf("%d", &sepA);
    printf("harga sepatu B adalah :");
    scanf("%d", &SepB);
    printf("sepatu A mendapat diskon 13%c ", d, &Dis13);
    hasil1 = sepA - (sepA * Dis13 / 100);
    printf("sehingga harganya menjadi : %d\n", hasil1);
    printf("sepatu B mendapat diskon 21%c ", d, &Dis21);
    hasil2 = SepB - (SepB * Dis21 / 100);
    printf("sehingga harganya menjadi : %d\n", hasil2);
    return 0;
}

File: Python/PRAK104-2210817110005-MIZZUDDINABDISSALAM.py
sepA = int(input("Harga sepatu A adalah    :"))
sepB = int(input("Harga sepatu B adalah    :"))
d1 = 0.13
h1 = sepA - (sepA * d1)
d2 = 0.21
h2 = sepB - (sepB * d2)

print("Sepatu A mendapat diskon 13% sehingga harganya menjadi ", round(h1))
print("Sepatu B mendapat diskon 21% sehingga harganya menjadi ", round(h2))